docusaurus/docs/Android/03-ui/05-components/05-message-list.mdx

```diff
@@ -461,4 +461,19 @@ If your predicate doesn't show avatars for your own messages (this is the defaul
 
 ```xml
 streamUiMessageEndMargin="0dp"
-```
\ No newline at end of file
+```
+## Configure giphy
+It is possible to configure the gifs size. Prior of version 4.23.0 gifs are autosizable and will resize thenselves accordingly with the size of the image they load. It is possible to use constant size gifs using `streamUiMediaAttachmentConstantSize` and set the desired height using `streamUiMediaAttachmentGiphyHeight`.
+
+As of v5.x.x, our Giphy holders are much more customizable. You can use three Giphy load modes:
+* `original`: The original Giphy size, as given by the API.
+* `fixedHeight`: Smaller size Giphy, that uses fixed height in order to improve scrolling performance and image loading speed.
+* `fixedHeightDownsampled`: Same as `fixedHeight`, but downsampled to only 6 frames, making it 6x smaller and faster to load, but less visually appealing.
+
```

Comment:
Let me know if this makes sense.

Literally - use the type for image quality and size (bytes).

Use Scale type for looks.